R/utils.R

```diff
@@ -0,0 +1,13 @@
+# #' @title Launch a call to function in the server and get the result
+# #'
+# #' @param datasources a list of \code{\link{DSConnection-class}} 
+# #' objects obtained after login. If the \code{datasources} argument is not specified
+# #' the default set of connections will be used: see \code{\link{datashield.connections_default}}.
```

Comment:
> If the \code{datasources} argument is not specified...

You don't need to write this for this function, assigning default connections to `datasources` is done in the `ds._` functions by
```
    if (is.null(datasources)) {
        datasources <- DSI::datashield.connections_find()
    }
```